ultralytics/engine/exporter.py

```diff
@@ -786,7 +780,7 @@ def write_calibration_cache(self, cache) -> None:
             # Load dataset w/ builder (for batching) and calibrate
             config.int8_calibrator = EngineCalibrator(
                 dataset=self.get_int8_calibration_dataloader(prefix),
-                batch=2 * self.args.batch,
+                batch=2 * self.args.batch,  # TensorRT INT8 calibration should use 2x batch size
```

Comment:
This is just removing the comment while keeping the code. But have you tested TRT INT8 export with this branch?

Reply:
This is the correct place to define the batch size. So I removed the comment from above and added it here!